src/EasyAbp.BookingService.EntityFrameworkCore/EasyAbp/BookingService/AssetCategories/AssetCategoryEfCoreQuerableExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Linq;
+using Microsoft.EntityFrameworkCore;
+
+namespace EasyAbp.BookingService.AssetCategories;
+
+public static class AssetCategoryEfCoreQueryableExtensions
+{
+    public static IQueryable<AssetCategory> IncludeDetails(this IQueryable<AssetCategory> queryable,
```

Comment:
The `Abp.Abp.Trees.EntityFrameworkCore` package has configured details, see:
https://github.com/EasyAbp/Abp.Trees/blob/master/src/EasyAbp.Abp.Trees.EntityFrameworkCore/EasyAbp/Abp/Trees/EfCoreTreeRepository.cs#L24-L25